package.json

```diff
@@ -116,6 +116,7 @@
     "jest": "^24.8.0",
     "lerna": "^3.16.4",
     "link-parent-bin": "^1.0.0",
+    "postcss-rtl": "1.6.0",
```

Comment:
why was this added?

Reply:
I think this has been added while merging..!